### PR TITLE
JBEAP-3572 Wicket 7.2 - wicket-war

### PR DESCRIPTION
**Pull request by OndraZizka**
Wicket was removed some time ago due to https://bugzilla.redhat.com/show_bug.cgi?id=1103713

### CODE REVIEW

**Comment by OndraZizka** [comment]
The wicket-ear is not upgraded, pls remove that before merging. TBD later.

**Comment by sgilda** [comment]
To avoid confusion, I am closing the pull request for now. 
1. It needs a JIRA associated with it.
2. The JIRA needs to be ACKed by the appropriate people (PM, QA, Dev)
3. I can not alter the pull. You need to remove the artifacts before you issue the pull request. 

We can reopen this if it looks like the JIRA is accepted.

**Comment by OndraZizka** [comment]
1) Jira in which project?
2) Which artifacts?

**Comment by OndraZizka** [comment]
1) https://issues.jboss.org/browse/JBEAP-3572
2) wicket-ear removed.

**Comment by martin-g** [comment]
Too bad this PR didn't make it!
@OndraZizka Is it a good idea to open a new ticket+PR so it could be included in the next release ?

File: wicket-ear/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+wicket-ear: Wicket Framework used in a WAR inside an EAR.
+=========================================================
+Author: Ondrej Zizka <ozizka@redhat.com>
+Level: Intermediate
+Technologies: Apache Wicket, JPA
+Summary: Demonstrates how to use the Wicket Framework 1.5 with the JBoss server using the Wicket-Stuff Java EE integration, packaged as an EAR
```
Comment (martin-g) [inline wicket-ear/README.md:6]: s/1.5/7.x/

File: wicket-ear/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Target Product: EAP
+Source: <https://github.com/jboss-jdf/jboss-as-quickstart/>
+
+What is it?
+-----------
+
+This is an example of how to use Wicket Framework 1.5 with JBoss AS, leveraging features of Java EE 6, using the Wicket-Stuff Java EE integration.
```
Comment (martin-g) [inline wicket-ear/README.md:13]: s/1.5/7.2/

File: wicket-ear/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        -`wicket-ear` - parent module
+            - `ejb`: Contains EJB beans and JPA entities. Creates a `.jar` file
+            - `war`: Contains the Wicket web application, which uses the EJB beans. Creates a `.war` file
+            - `ear`: Packages the EJB JAR and WAR into an EAR. Creates an `.ear` file
+
+
+System requirements
+-------------------
+
+All you need to build this project is Java 6.0 (Java SDK 1.6) or better, Maven 3.0 or better.
```
Comment (martin-g) [inline wicket-ear/README.md:32]: Java 7

**Comment by martin-g** [comment]
The PR title says Wicket 7.2 but the versions in the pom.xml say 1.5.5, and it uses net.ftlines.wicket.cdi.
It seems this PR is not quite up to date.

**Comment by sgilda** [comment]
I believe we suggested this pull request target the wildfly quickstarts for now. It came in too late to be QAd for this release.

**Comment by martin-g** [comment]
Yes, I saw this comment but since both the JIRA ticket and the PR are closed (without links to new ones) I have the feeling that it will be forgotten.

**Comment by klopfdreh** [comment]
+1 for a new PR / Ticket with the new version of Wicket.

**Comment by sgilda** [comment]
I cloned the JIRA to track adding it to the WildFly quickstarts: https://issues.jboss.org/browse/WFLY-6432

**Comment by martin-g** [comment]
Thank you!

**Comment by klopfdreh** [comment]
Thanks from me, too!

**Comment by sgilda** [comment]
Any time! :-)

**Pull request closed without merging**